safe-defaults/tmpfiles-enable.conf.in

```diff
@@ -6,7 +6,8 @@
 # if a symlinked config file in the ostree is no longer needed it should
 # be made empty rather than deleted, otherwise the change from L to r
 # in this file will be ineffective.
-L @SYSCONFDIR@/NetworkManager/conf.d/opendns-family-shield.conf - - - - @DATAROOTDIR@/eos-safe-defaults/nm-opendns-family-shield.conf
```

Comment:
We need an `r` line for `@SYSCONFDIR@/NetworkManager/conf.d/opendns-family-shield.conf` in this file. See the comment right above this line, and the `r`s at the bottom of the file.

Reply:
Ah!  I used `mv`, instead of `rm` when I wrote the configuration, which did not trigger the condition in my brain.  I shall update the rule in my brain again.